_docs_use-cases/use-case-transport-create-relationship-with-relationshiptemplate.md

```diff
@@ -42,12 +42,12 @@ api_route_regex: ^POST /api/v2/Relationships$
 
 {% include properties_list.html %}
 
-This use case intends to create a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) based on a received [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
+This use case intends to create a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) based on a received [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate). The Relationship will be with the Template's creator.
```

Comment:
Please write "RelationshipTemplate" instead of "Template" or use "Templator" to refer to the creator of the RelationshipTemplate as we do in other guides.

Reply:
Templator is not used in the use cases though